django/cantusdb_project/templates/base.html

```diff
@@ -37,9 +37,10 @@
             function handleLogButtonClick() {
```

Comment:
Is is possible to remove this piece of js all together? If I understand correctly, with the current design, we only need this button as a hyperlink to the login page, while logout will be handled by another hyperlink.

Reply:
I think we still need to keep this around because the "Log in" button uses this. Unless you want the "Log in" button to be replaced with a hyperlink.